numba_dppy/tests/integration/test_dpnp_interop.py

```diff
@@ -44,37 +44,27 @@ def usm_type(request):
     return request.param
 
 
-def test_dpnp_create_array_in_context(offload_device, dtype):
-    if not ensure_dpnp():
-        pytest.skip("No DPNP")
-
+@pytest.mark.parametrize("filter_str", filter_strings)
+def test_dpnp_create_array_in_context(filter_str, dtype):
     import dpnp
 
-    if skip_test(offload_device):
-        pytest.skip("No device for " + offload_device)
-
     if (
         "opencl" not in dpctl.get_current_queue().sycl_device.filter_string
-        and "opencl" in offload_device
+        and "opencl" in filter_str
```

Comment:
What was the reason to rename variable?

Reply:
Variable is renamed because decorator **@pytest.mark.parametrize("filter_str", filter_strings)** was added

Reply:
First argument could be any string containing variable name you will expect in test function arguments. So you can select any name. Use this possibility to select previous name.